frontend/src/app/main/ui/workspace/tokens/components/controls/input_tokens_value.cljs

```diff
@@ -24,12 +23,13 @@
    [:error {:optional true} :boolean]
    [:value {:optional true} :string]])
 
-(mf/defc input-tokens*
+(mf/defc input-tokens-value*
   {::mf/props :obj
    ::mf/forward-ref true
-   ::mf/schema schema::input-tokens}
-  [{:keys [class label id max-length error value children] :rest props} ref]
-  (let [ref   (or ref (mf/use-ref))
+   ::mf/schema schema::input-tokens-value}
+  [{:keys [class label max-length error value children] :rest props} ref]
+  (let [id (mf/use-id)
+        ref   (or ref (mf/use-ref))
```

Comment:
This `mf/use-ref` is on conditional, breaks the hooks rule (all hooks should be defined statically)

I known this is not your code, but this should be fixed.

Reply:
I'm not sure what is the bet way to solve this. Please check my last commit to see my proposal. Otherwise, lets see what would be the best approach together.